File: experiment/ifelse_earlyreturn/earlyreturn.rb
def letter_for_number num
  return '-' if(num < 1 || num > 26)
  letters = "abcdefghijklmnopqrstuvwxyz";
  letters_arr = letters.split('') # ['a', 'b', ..]
  return letters_arr[num - 1]
end

result = letter_for_number 3

File: experiment/ifelse_earlyreturn/earlyreturn.py
def letter_for_number(num):

  if(num < 1 or num > 26):
    return '-'

  letters = "abcdefghijklmnopqrstuvwxyz"
  letters_arr = list(letters)  # ['a', 'b', ..]
  return letters_arr[num - 1]

result = letter_for_number(3)
